doc/sphinx/changes.rst

```diff
@@ -8,6 +8,753 @@ Recent changes
+  by Gaëtan Gilbert).
+- **Added:**
+  option `-time-file`, like `time` but outputting to a file
+  (`#17430 <https://github.com/coq/coq/pull/17430>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  :cmd:`Validate Proof` runs the type checker on the current proof,
+  complementary with :cmd:`Guarded` which runs the guard checker.
+  (`#17467 <https://github.com/coq/coq/pull/17467>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  :attr:`clearbody` for :cmd:`Let` to clear the body of a let-in in an interactive
+  proof without kernel enforcement.  (This is the behavior that was previously
+  provided by using :cmd:`Qed`, which is now deprecated for `Let`\s.)
+  (`#17544 <https://github.com/coq/coq/pull/17544>`_,
+  by Gaëtan Gilbert).
+- **Deprecated:**
+  Using :cmd:`Qed` with :cmd:`Let`. End the proof with :cmd:`Defined` and use :attr:`clearbody`
+  instead to get the same behavior
+  (`#17544 <https://github.com/coq/coq/pull/17544>`_,
+  by Gaëtan Gilbert).
+- **Fixed:**
+  universe monomorphic inductives and records do not ignore :flag:`Universe Minimization ToSet`
+  (`#17285 <https://github.com/coq/coq/pull/17285>`_,
+  fixes `#13927 <https://github.com/coq/coq/issues/13927>`_,
+  by Gaëtan Gilbert).
+
+Command-line tools
+^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Do not pass the ``-rectypes`` flag by default in
+  ``coq_makefile`` when compiling OCaml code, since
+  it is no longer required by Coq. To re-enable passing
+  the flag, put ``CAMLFLAGS+=-rectypes`` in the local makefile,
+  e.g., ``CoqMakefile.local`` (see :ref:`coqmakefilelocal`)
+  (`#17038 <https://github.com/coq/coq/pull/17038>`_,
+  by Karl Palmskog with help from Gaëtan Gilbert).
+- **Changed:**
+  disable inclusion of variable binders in coqdoc indexes by default,
+  and provide a new coqdoc option `--binder-index` for including them
+  (`#17045 <https://github.com/coq/coq/pull/17045>`_,
+  fixes `#13155 <https://github.com/coq/coq/issues/13155>`_,
+  by Karl Palmskog).
+- **Added:**
+  `coqdoc` handles multiple links to the same source. For
+  example when declaring an inductive type `t` all occurences
+  of `t` itself and its elimination principles like `t_ind`
+  point to its declaration
+  (`#17118 <https://github.com/coq/coq/pull/17118>`_,
+  by Enrico Tassi).
+- **Added:**
+  Command line options :n:`-require lib` (replacing
+  :n:`-load-vernac-object lib`) and :n:`-require-from root lib`
+  respectively equivalent to vernacular commands :n:`Require lib` and
+  :n:`From root Require lib`
+  (`#17364 <https://github.com/coq/coq/pull/17364>`_, by Hugo Herbelin).
+- **Added:**
+  `coqtimelog2html` command-line tool used to render the timing files produced with `-time`
+  (which is passed by `coq_makefile` when environment variable `TIMING` is defined)
+  (`#17411 <https://github.com/coq/coq/pull/17411>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  option `-timef`, like `time` but outputting to a file
+  (`#17430 <https://github.com/coq/coq/pull/17430>`_,
+  by Gaëtan Gilbert).
```

Comment:
This changelog entry is duplicated and incorrect. It appears both in "Commands and options" and in "Command-line tools". The latter is the more correct location, but the actual content of the changelog entry is incorrect here, because the option was renamed. This was signaled on the PR itself after it was merged: https://github.com/coq/coq/pull/17430/files/3d982ac1d0aaddff27d52e8e775bf2a1963ca0b0#r1152380804